File: gateOne/Day Four/java/TaskOne.java
import java.util.Scanner;

	public class TaskOne {

		public static void main(String [] args){
		Scanner input = new Scanner(System.in);

		System.out.println("what is your name");
		String name = input.nextLine();

		String [] []  ticToe = new String [3][3];

		String user;
		for(int row = 0 ; row <= 3; row ++){
		
			for(int column = 0 ; column <= 3 ; column++){

		System.out.println("enter the alphabet between x and 0");
		user = input.nextLine().trim().toLowerCase();
		

	if(user.equals(("x")) || user.equals(("o"))){
	ticToe[row][column] = user;
	}
      
    

else {

System.out.println("invalid input pls put in the right own");
ticToe[row][column] = user;
 
   }
  
  }
}

for(int row = 0 ; row <= 3 ; row ++){
	for(int column = 0 ; column <= 3 ; column++){

 	System.out.println(ticToe[row][column]);
        }
      }
    }
   }

File: gateOne/Day Four/phython/TaskOne.py
user_name = str(input("what is your name"))
words = [9]
for word in range(0 ,3):
	for letters in range(0 ,3):
		user_input = str(input("enter the alphabet x or o"))
		
	if user_input == "x" and user_input == "o":
		words[word][letters] = user_input
	else:
		print('invalid input pls put in the correct own')
		words[word][letters] = user_input

for word in range(0 ,3):
	for letters in range(0 ,3):
		print(number[word][letters])